src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -439,7 +439,7 @@ def transform_summary(self,
         es_response = es_search.execute(ignore_cache=True)
         self._translate_response_aggs(es_response)
         final_response = SummaryResponse(es_response.to_dict())
-        if logger.isEnabledFor(logging.DEBUG):
+        if logger.isEnabledFor(logging.DEBUG and config.debug == 2):
```

Comment:
```suggestion
        if logger.isEnabledFor(logging.DEBUG) and config.debug == 2:
```

Do you test you changes?